cdk/src/open-data-platform/data-plane/api/api-stack.ts

```diff
@@ -0,0 +1,64 @@
+import { Duration } from 'aws-cdk-lib';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as lambda from 'aws-cdk-lib/aws-lambda-nodejs';
+import * as apigateway from 'aws-cdk-lib/aws-apigateway';
+import { Construct } from 'constructs';
+import { SchemaProps } from '../schema/schema-props';
+import * as path from 'path';
+
+export class ApiStack extends Construct {
+  constructor(scope: Construct, id: string, props: SchemaProps) {
+    super(scope, id);
+
+    const { cluster, vpc, db, credentialsSecret } = props;
+
+    const lambdaFunction = new lambda.NodejsFunction(this, 'geolocate-handler', {
+      entry: `${path.resolve(__dirname + '/../../../../../')}/api/src/geolocate/get.handler.ts`,
```

Comment:
It feels like there should be a better way to do this but the absolute path wouldn't work here

Reply:
https://nodejs.org/api/process.html#processcwd might help here.

Reply:
Thank you!